terraform/rtmp-server/nginx.conf

```diff
@@ -18,11 +18,7 @@ rtmp {
       deny publish all;
       drop_idle_publisher 10s;
       allow play all;
-      hls on;
-      hls_path /root/hls;
-      hls_sync 100ms;
-      hls_fragment 10s;
-      hls_playlist_length 60m;
```

Comment:
@ASoTNetworks so what were all these for before?

Reply:
It was a copy of his config where he actually hosts HLS stuff on the server. We are not.

Reply:
Those were for HLS that is not needed on the rtmp server since IPFS server pulls the RTMP stream.

Reply:
Looks like I can delete the `mkdir /root/hls` somewhere too